### ISSUE TITLE
docker image on linux still has permissions issues

### ISSUE THREAD

**Issue by jlevon** [marp-team/marp-cli#360]
### Version of Marp Tool

latest marp-cli docker image

### Operating System

Linux

### Environment

Running docker on Ubuntu

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

### How to reproduce

```
$ docker run --rm --init -v $PWD:/home/marp/app/ -e MARP_USER="$(id -u):$(id -g)" -e LANG=$LANG marpteam/marp-cli --pdf --allow-local-files -o ioregionfd.pdf ioregionfd.md
[  INFO ] Converting 1 markdown...
[  WARN ] Insecure local file accessing is enabled for conversion from
          ioregionfd.md.
[ ERROR ] Failed converting Markdown. (EACCES: permission denied, open
          '/tmp-7-DArfLrvtsb2d-.html')
```

There's no mention of this issue on the hub page.

### Expected behavior

I can follow the instructions and build OK.

### Actual behavior

EACCES on writing to / inside the container

**Comment by yhatt** [comment]
It probably a permission trouble for writing intermediate HTML file. What would happen if run without `--allow-local-files`? (It will convert Marp slide without file system)

CC: @davebaird (https://github.com/marp-team/marp-cli/pull/334)

**Comment by jlevon** [comment]
You're right, it works without that, thanks. Is there a way to not use that flag, but still include images (for example)

**Comment by yhatt** [comment]
You still can use online images, just like `![](https://example.com/xxxxxx.jpg)`.

**Comment by jlevon** [comment]
Thanks, but that's an issue for content that needs to be behind a VPN etc.

**Comment by yhatt** [comment]
Fixed by #379.
